python/functional-programming/sum-all-map-functions/the-sum-built-in.md

```diff
@@ -87,7 +86,5 @@ print(sum1 if ??? > ??? else sum2)
 ---
 
 ## Footnotes
-
-[1:Alternatives to `sum`]
-*An example of a rare case where an alternative method is superior to `sum` is when adding floating point values. When precision is important, it is better to use `math.fsum()` which deliberately avoids loss of precision. For more information, see the Python Docs in the Learn More section.*
- 
+[1: Why is the result a float?]
+Notice how the result of our `sum` is of type `float` as opposed to `int` since Python automatically converts `float` and `int` addition results to `float`s to avoid loss of precision. This is not specific to `sum` but happens whenever we do arithmetic operations on mixed number types.
```

Comment:
````suggestion
[1: Why is the result a float?]
Notice how the result of our `sum` is of type `float` as opposed to `int` since Python automatically converts `float` and `int` addition results to `float`s to avoid loss of precision. 

This is not specific to `sum` but happens whenever we do arithmetic operations on mixed number types.

```py
2 + 3.0 # 5.0
```
````